### PR TITLE
`azurerm_spring_cloud_customized_accelerator` - use pandora sdk

### PR DESCRIPTION
**Pull request by ms-henglu**
This PR switches to Pandora SDK for `azurerm_spring_cloud_customized_accelerator` resource.

### CODE REVIEW

File: internal/services/springcloud/spring_cloud_customized_accelerator_resource.go [hunk 1]
```diff
@@ -396,88 +400,78 @@ func (s SpringCloudCustomizedAcceleratorResource) Delete() sdk.ResourceFunc {
...
 			return nil
 		},
 	}
 }
 
-func expandSpringCloudCustomizedAcceleratorGitRepository(repository []GitRepositoryModel) *appplatform.AcceleratorGitRepository {
+func expandSpringCloudCustomizedAcceleratorGitRepository(repository []GitRepositoryModel) appplatform.AcceleratorGitRepository {
 	if len(repository) == 0 {
-		return nil
+		return appplatform.AcceleratorGitRepository{}
```
Comment (magodo) [inline internal/services/springcloud/spring_cloud_customized_accelerator_resource.go:422]: Will this set an empty json object in the request body when the repository is not specified?

**Review by magodo — CHANGES_REQUESTED** [review]
Thanks for this PR!
I've taken a look through and left some comments inline, but this is mostly looking good to me 👍

File: internal/services/springcloud/spring_cloud_customized_accelerator_resource.go (hunk 1 above)
Comment (ms-henglu) [reply internal/services/springcloud/spring_cloud_customized_accelerator_resource.go:422]: This is a required block and has required nested field, actually this if-statement won't be executed.

**Review by katbyte — APPROVED** [review]
LGTM 🌴

**Pull request merged**